articles/multifactor-authentication/developer/step-up-with-acr.md

```diff
@@ -0,0 +1,76 @@
+---
```

Comment:
I would expect first to see something about "How to check if the user has logged in with a second factor" and explain that you need to check the acr claim in the id_token. You don't need to verify the token to do that, that should have already happened as part of the login process in your API or regular web app (assuming you are checking this on the server side).